wurst/util/StringUtils.wurst

```diff
@@ -17,6 +18,7 @@ int array width
 constant MAX_INDEX = '~' + 1
 constant DEFAULT_WIDTH = 0
 constant DEFAULT_STRING = ""
+constant let	 WC3LIMIT_OBJECTS_DESCRIPTION_LENGTH = 1023 // Should be a global for whole WurstScript or StdLib at least
```

Comment:
why is there a tab after "let" ?

Reply:
oh cause when I had the other constants, I tabbed them to start at same place... Nevermind, gonna change it.